portfolio/src/main/webapp/script.js

```diff
@@ -26,6 +26,70 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function expandDesign(clickedId){
+//whenever you click an image on the description page, this pulls up more images in the category alongside descriptions
+const designPage = document.getElementById('design_container');
+designPage.innerHTML = "";
+console.log(clickedId); //REMOVE WHEN DONE
```

Comment:
if this commit is up for review, logs like this should be removed. if you didn't mean for me to see this yet though (and it just came up here because of some branching stuff) then that's totally fine. Just be sure to remove it before pushing to master.

Reply:
yea I am working on a new pull request that fixes all of these things. None of this function was supposed to be up yet since I completely restructured a lot of things after the initial commits.